test/e2e/e2e_test.go

```diff
@@ -441,12 +446,17 @@ func TestE2EFlow(t *testing.T) {
 		VerifyResourcesProvisionedForSignup(t, awaitilities, johnExtraSignup, "deactivate30", "base")
 
 		// check if the MUR and UA counts match
-		currentToolchainStatus := hostAwait.WaitForToolchainStatus(t,
+		originalUserAccountCounts := make(map[string]int, len(originalToolchainStatus.Status.Members))
+		for _, m := range originalToolchainStatus.Status.Members {
+			originalUserAccountCounts[m.ClusterName] = m.UserAccountCount
+		}
+		hostAwait.WaitForToolchainStatus(t,
 			wait.UntilToolchainStatusHasConditions(ToolchainStatusReadyAndUnreadyNotificationNotCreated()...),
 			wait.UntilToolchainStatusUpdatedAfter(time.Now()),
-			wait.UntilHasMurCount("external", originalMursPerDomainCount["external"]+8))
-		VerifyIncreaseOfUserAccountCount(t, originalToolchainStatus, currentToolchainStatus, johnsmithMur.Spec.UserAccounts[0].TargetCluster, 8)
-		VerifyIncreaseOfUserAccountCount(t, originalToolchainStatus, currentToolchainStatus, targetedJohnMur.Spec.UserAccounts[0].TargetCluster, 1)
+			wait.UntilToolchainStatusHasMurCount("external", originalMursPerDomainCount["external"]+8),
+			wait.UntilToolchainStatusHasUserAccountCount(johnsmithMur.Spec.UserAccounts[0].TargetCluster, originalUserAccountCounts[johnsmithMur.Spec.UserAccounts[0].TargetCluster]+7),
```

Comment:
that's what I needed to change on my temp cluster (which is confusing, I admit), but I did not take time to check on OpenShift CI yet

Reply:
agree, not related to your changes of course, but it's not obvious on what is the amount of UserAccountCounts (soon Spaces) to expect in this test, since there are multiple places in the parent and subtests were signups are created/deleted.

Reply:
yes, exactly :/

Reply:
Your changes shouldn't affect the counts so there could be something wrong somewhere but I'm not sure where 🤔 . I think the changes related to `VerifyIncreaseOfUserAccountCount` and related functions should be reverted and done in a separate PR since they aren't directly related to this PR's goal and are not working as expected.

Reply:
agreed, there should be no change in the counts, I'm going to look at it